app/assets/javascripts/cascade/level/homepage-masthead.js

```diff
@@ -8,7 +9,35 @@ $(function () {
         ieObjectFitFallback();
     }
 });
-
+function fetchCuratorImages() {
+    $.ajax({
+        url: 'https://api.curator.io/v1/feeds/ef183959-c3ad-4f2d-b90e-390c5d766fac/posts?api_key=11a4445f-6005-4040-9ff2-fd90d3aaa8a6',
+        type: 'GET',
+        success: manipulateCuratorImages,
+        error: function (data, status, error) {
```

Comment:
is there any fallback for the images if they failed to load? what does it look like

Reply:
@corejh It'll fallback/reveal the hardcoded images in https://github.com/chapmanu/cascade-assets/pull/677/files#diff-3557a4980111b7b0b76b6edd45fdc5b3

demo: http://recordit.co/4GCFClsUbo